builtin/providers/aws/resource_aws_alb_test.go

```diff
@@ -71,6 +71,7 @@ func TestAccAWSALB_basic(t *testing.T) {
 					resource.TestCheckResourceAttrSet("aws_alb.alb_test", "zone_id"),
 					resource.TestCheckResourceAttrSet("aws_alb.alb_test", "dns_name"),
 					resource.TestCheckResourceAttrSet("aws_alb.alb_test", "arn"),
+					resource.TestCheckResourceAttrSet("aws_alb.alb_test", "ip_address_type"),
```

Comment:
Any specific reason we can't check the value here and verify that it defaults to `ipv4`?